tests/lava/proc/monitor/test_monitors.py

```diff
@@ -342,6 +342,141 @@ def test_proc_params_accessible_in_proc_model(self):
         self.assertEqual(next(iter(exe.py_builders)).proc_params,
                          monitor.proc_params)
 
+    # New API unittests
+    def test_single_recorder_with_single_targer(self):
```

Comment:
The name should reflect that you are monitoring a Port. There should then be another unit test for monitoring an individual Var.